arcgis-ios-sdk-samples/Display information/Create symbol styles from web styles/CreateSymbolStylesFromWebStylesViewController.swift

```diff
@@ -0,0 +1,306 @@
+            return nil
+        } else {
+            let createSwatchOperation = symbol.createSwatch { [weak self] (image, _) in
+                guard let self = self else { return }
+                self.imageOperations[indexPath] = nil
+                if let image = image {
+                    self.cachedImages[indexPath] = image
+                    self.tableView?.reloadRows(at: [indexPath], with: .automatic)
+                }
+            }
+            imageOperations[indexPath] = createSwatchOperation
+            return nil
+        }
+    }
+}
+
+// MARK: - SymbolCategory
+
+private enum SymbolCategory: CaseIterable, Comparable {
+    case atm, beach, campground, cityHall, hospital, library, park, placeOfWorship, policeStation, postOffice, school, trail
+    
+    /// The names of the symbols in the web style.
+    var symbolName: String {
+        let name: String
+        switch self {
+        case .atm:
+            name = "atm"
+        case .beach:
+            name = "beach"
+        case .campground:
+            name = "campground"
+        case .cityHall:
+            name = "city-hall"
+        case .hospital:
+            name = "hospital"
+        case .library:
+            name = "library"
+        case .park:
+            name = "park"
+        case .placeOfWorship:
+            name = "place-of-worship"
+        case .policeStation:
+            name = "police-station"
+        case .postOffice:
+            name = "post-office"
+        case .school:
+            name = "school"
+        case .trail:
+            name = "trail"
+        }
+        return name
+    }
+    
+    /// The category names of features represented by a type of symbol.
+    var symbolCategoryValues: [String] {
+        let values: [String]
+        switch self {
+        case .atm:
+            values = ["Banking and Finance"]
+        case .beach:
+            values = ["Beaches and Marinas"]
+        case .campground:
+            values = ["Campgrounds"]
+        case .cityHall:
+            values = ["City Halls", "Government Offices"]
+        case .hospital:
+            values = ["Hospitals and Medical Centers", "Health Screening and Testing", "Health Centers", "Mental Health Centers"]
+        case .library:
+            values = ["Libraries"]
+        case .park:
+            values = ["Parks and Gardens"]
+        case .placeOfWorship:
+            values = ["Churches"]
+        case .policeStation:
+            values = ["Sheriff and Police Stations"]
+        case .postOffice:
+            values = ["DHL Locations", "Federal Express Locations"]
+        case .school:
+            values = ["Public High Schools", "Public Elementary Schools", "Private and Charter Schools"]
+        case .trail:
+            values = ["Trails"]
+        }
+        return values
+    }
+    
+    static func < (lhs: SymbolCategory, rhs: SymbolCategory) -> Bool {
+        return lhs.symbolName < rhs.symbolName
+    }
```

Comment:
This is unnecessary as the compiler will [automatically synthesize](https://github.com/apple/swift-evolution/blob/main/proposals/0266-synthesized-comparable-for-enumerations.md) an implementation.

Reply:
😮  Wow. Nice to learn!